src/js/components/jtable/TableRow.jsx

```diff
@@ -13,6 +14,7 @@ type Props = {
     className?: string,
     children?: ReactChildren,
     href?: string,
+    linkTo?: string,
```

Comment:
@sophistifunk I like the functionality but I am a little worried about coupling the JDL to `react-router`.

What if we introduced a `link` prop of `PropTypes.element` that allowed the client code to pass in a React element used to wrap the row? We should just be able to `React.clone` the `link` and then pass the `TableRow` as its children?